cmd/juju/model/addgeneration_test.go

```diff
@@ -80,22 +81,19 @@ func (s *AddGenerationSuite) TestRunCommand(c *gc.C) {
 	mockController, mockAddGenerationCommandAPI := setUpMocks(c)
 	defer mockController.Finish()
 
-	mockAddGenerationCommandAPI.EXPECT().AddGeneration().Return(nil).Times(1)
+	mockAddGenerationCommandAPI.EXPECT().AddGeneration(gomock.Any()).Return(nil).Times(1)
 
 	ctx, err := s.runCommand(c, mockAddGenerationCommandAPI)
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(cmdtesting.Stdout(ctx), gc.Equals, "target generation set to next\n")
 }
 
 func (s *AddGenerationSuite) TestRunCommandFail(c *gc.C) {
-	c.Skip("Until apiserver ModelGeneration.AddGeneration() implemented")
-
 	mockController, mockAddGenerationCommandAPI := setUpMocks(c)
 	defer mockController.Finish()
 
-	mockAddGenerationCommandAPI.EXPECT().AddGeneration().Return(errors.Errorf("failme")).Times(1)
+	mockAddGenerationCommandAPI.EXPECT().AddGeneration(gomock.Any()).Return(errors.Errorf("failme")).Times(1)
```

Comment:
Implicit times(1).

Reply:
removed.  found the other changes in generation cmd test code too.